IPython/core/historyapp.py

```diff
@@ -10,9 +10,15 @@
 
 from traitlets.config.application import Application
 from .application import BaseIPythonApplication
-from traitlets import Bool, Int, Dict
+from traitlets import Bool, Int, Dict, Unicode, CaselessStrEnum
 from ..utils.io import ask_yes_no
 
+show_hist_help = """Show recent entries from the IPython history database.
+
+Limited to the last 1000 unless `--limit=N` is supplied; `--limit=0` or `-1`
+prints all entries in the database.
+"""
+
```

Comment:
You can inline this directly if you wish, you don't have to rely on a seperate variable. Up to you.